poly/main.go

```diff
@@ -50,28 +50,28 @@ func run(args []string) {
 
 // application is a function that defines instances of our app. it's where we template commands and where initial arg parsing occurs.
 func application() *cli.App {
+	inputFlag := cli.StringFlag{
+		Name:  "i",
+		Usage: "Specify file input type. Options are gff, gbk/gb, fasta, and json. For use with pipes.",
+	}
+	outputFlag := cli.StringFlag{
+		Name:  "o",
+		Value: "json",
+		Usage: "Specify file output type or path. Defaults to json.",
+	}
```

Comment:
I noticed that the input and output flags were the same for the top level and the sub commands.

I consolidated them here and standardized the help text.